main/speech/TTS.py

```diff
@@ -25,15 +26,17 @@ def speak_flite_tts(text):
     :param text: Text which is needed to be spoken
     :return: None
     """
-    filename = '.response'
-    with open(filename, 'w') as f:
-        f.write(text)
-    # Call flite tts to reply the response by Susi
-    flite_speech_file = config['flite_speech_file_path']
-    logger.debug('flite -voice file://%s -f %s', flite_speech_file, filename)
-    subprocess.call(   # nosec #pylint-disable type: ignore
-        ['flite', '-v', '-voice', 'file://' + flite_speech_file, '-f', filename, '-o', 'extras/output.wav'])   # nosec #pylint-disable type: ignore
-    subprocess.call(['play', 'extras/output.wav'])   # nosec #pylint-disable type: ignore
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        fd, filename = tempfile.mkstemp(text=True, dir=tmpdirname)
+        with open(fd, 'w') as f:
+            f.write(text)
+        # Call flite tts to reply the response by Susi
+        flite_speech_file = config['flite_speech_file_path']
+        logger.debug('flite -voice file://%s -f %s', flite_speech_file, filename)
+        fdout, wavOutput = tempfile.mkstemp(suffix='.wav', dir=tmpdirname)
```

Comment:
Could you please try to follow Python coding style in naming variable? (prefer `lower_case_thing` to `someThing`).

Reply:
Fixed in commit 443a45700fba7438983aa98cb09fc63426e9a64f